static/js/src/public/store/charms.js

```diff
@@ -21,15 +23,21 @@ class Charms {
           return;
         }
 
+        this.groupAllCharms();
+
         this.filterCharms();
         this.handleShowAllCharmsButton();
         this.handleFilterButtonMobileOpenClick();
-        this.handleFilters();
+        this.handleFilterCLick();
```

Comment:
can you lowercase the `L` in `handleFilterCLick`?

Reply:
of course I can, but I will think if I want to do it lol